io.github.nyre221.kiview.json

```diff
@@ -0,0 +1,45 @@
+{
+   "base": "io.qt.qtwebengine.BaseApp",
+   "base-version": "6.6",
+   "command": "kiview",
+   "build-options": {
+    "env": [
+      "QTWEBENGINE_DISABLE_SANDBOX=1"
+    ]
+  },
+
+   "finish-args": [
+      "--share=ipc",
+      "--device=dri",
+      "--socket=fallback-x11",
+      "--socket=wayland",
+      "--env=QML_IMPORT_PATH=/app/qml" ,
+      "--socket=pulseaudio",
+      "--socket=session-bus",
+      "--filesystem=host:ro",
+      "--filesystem=/tmp",
+      "--talk-name=org.freedesktop.Flatpak",
+      "--talk-name=org.kde.klipper",
+      "--talk-name=org.freedesktop.DBus",
```

Comment:
why do you need `org.freedesktop.DBus`.

Reply:
It's there from a previous attempt to not use `--socket=session-bus`
I can remove it

Reply:
removed